ports/cxd56/common-hal/pulseio/PulseOut.c

```diff
@@ -59,7 +59,7 @@ static bool pulseout_timer_handler(unsigned int *next_interval_us, void *arg)
 }
 
 void common_hal_pulseio_pulseout_construct(pulseio_pulseout_obj_t* self,
-                                            const pulseio_pwmout_obj_t* carrier,
+                                            const pwmio_pwmout_obj_t* carrier,
```

Comment:
This file is different than the other PulseOut.c files in common-hal, in that it appears to access the pulseio file include through `"shared-bindings/pulseio/PulseOut.h"`. Should we make that consistent across all files, or add in a direct reference to pulseio in the spirit of include what you use?

Could be revisited as part of a more general inclusion style review.

Reply:
Ya, generally it's better to include what you use but ensuring that in a manual way is impossible. I think this is fine for now.